acra/configuring-maintaining/optimisations/connection_reuse.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Connection reuse
+weight: 6
+---
+
+# Connection reuse
+
+Most of libraries/frameworks for work with SQL databases have a dedicated object for the connection with database.
+The one you get after soccessful connetion to database with used credentials.
+It is recommended to **reuse** such object as much as possible.
+
+Reusing connections will result in lower latencies (no need to wait for connection before every SQL request).
+
+It will also reduce the load on AcraServer and the database behind since they won't have to handle new connections so frequently.
```

Comment:
we can say that it will reduce amount of memory that AcraServer uses to handle connections. Because it works similar to any database that allocate temporary buffers for each new connection. AcraServer does it too.

Reply:
Changed, is new version better?